### PR TITLE
Renames instabitulari to syntflesh

### PR DESCRIPTION
**Pull request by Time-Green**
###  Why It's Good For The Game
Everyone calls it synthflesh. The wiki and code both also refer to it as Synthflesh. Instabitulari is called the way it us to indicate that it heals both brute and burn. While nice, there's plenty of chemicals that also ignore this naming convention in exchange for using a more trivial name (Miner's salve, Saline-glucose solution). Instabitulari is in creation and application synthethic flesh. Apart from brute and burn healing, it can also undo husking by refreshing the skin with synthethic flesh. 

:cl:
tweak: Instabitulari is called synthflesh again
/:cl:

### CODE REVIEW

**Comment by Jared-Fogle** [comment]
no please i just learned how to spell it

File: code/game/machinery/limbgrower.dm [hunk 1]
```diff
@@ -217,12 +217,12 @@
...
 /obj/machinery/limbgrower/proc/can_build(datum/design/D)
-	return (reagents.has_reagent(/datum/reagent/medicine/c2/instabitaluri, D.reagents_list[/datum/reagent/medicine/c2/instabitaluri]*prod_coeff)) //Return whether the machine has enough instabitaluri to produce the design
+	return (reagents.has_reagent(/datum/reagent/medicine/c2/synthflesh, D.reagents_list[/datum/reagent/medicine/c2/synthflesh]*prod_coeff)) //Return whether the machine has enough synthflesh to produce the design
 
 /obj/machinery/limbgrower/proc/get_design_cost(datum/design/D)
 	var/dat
-	if(D.reagents_list[/datum/reagent/medicine/c2/instabitaluri])
-		dat += "[D.reagents_list[/datum/reagent/medicine/c2/instabitaluri] * prod_coeff] SynthFlesh (instabitaluri)"
+	if(D.reagents_list[/datum/reagent/medicine/c2/synthflesh])
+		dat += "[D.reagents_list[/datum/reagent/medicine/c2/synthflesh] * prod_coeff] SynthFlesh (synthflesh)"
```
Comment (Jared-Fogle) [inline code/game/machinery/limbgrower.dm:225]:
```suggestion
		dat += "[D.reagents_list[/datum/reagent/medicine/c2/synthflesh] * prod_coeff] Synthflesh"
```

File: html/changelogs/.all_changelog.yml [hunk 2]
```diff
@@ -33707,9 +33707,9 @@ DO NOT EDIT THIS FILE BY HAND!  AUTOMATICALLY GENERATED BY ss13_genchangelog.py.
       mob.
   MacBlaze1:
   - tweak: Changed the eye damage of aiuri to .25 from 1, does not change healing.
-  - tweak: Changed toxin conversion of instabitaluri to 66% from 75%, does not buff
```
Comment (Jared-Fogle) [inline html/changelogs/.all_changelog.yml:33710]: Probably don't need to change the changelogs.

**Comment by ExcessiveUseOfCobblestone** [comment]
Combo healers either need a distinct suffix or it needs to be instab.

The chems you mentioned weren’t touched in function (yet?) so their names have not needed to go to the convention.

File: html/changelogs/.all_changelog.yml (hunk 2 above)
Comment (Time-Green) [reply html/changelogs/.all_changelog.yml:33710]: Woops, I thought I double checked

**Comment by Time-Green** [comment]
> Combo healers either need a distinct suffix or it needs to remain instabitaluri.
> 
> The chems you mentioned weren’t touched in function (yet?) so their names have not needed to go to the convention but synthetic chems should be abiding by this.
> 
> I am still fine with the “brand” (convention) style although it is a bit of a pain given how the RCs work off that name.

I understand wanting everything to be accurately named, but even in real life there are exceptions. No one in their right mind calls water dihydrogenmonoxide. There's pages filled with chemicals that use a trivial name because it's easier for everyone. 

If it's called synthflesh by convention everywhere, then why even bother calling it instabitulari? A chemist is gonna read instabitulari when checking contents and is going to have to enter instabitulari when entering it in machinery. We could make exceptions everywhere to support two names, but why? 

It's just a headache for everyone and people aren't really gaining anything from having it called instabitulari. In my opinion, the name synthflesh is even a better descriptor since it heals all flesh damage (brute, burn, husking).

**Comment by Time-Green** [comment]
@ExcessiveUseOfCobblestone alternatively we call the brute, burn and husk healing hybrid suffix 'flesh' so we can call it synthflesh. Thereby we follow the naming convention while also being called synthflesh

**Comment by TemporalOroboros** [comment]
-flesh is a bit on the nose, but last I checked -esh wasn't taken.

**Comment by stylemistake** [comment]
Just use whatever name sticks to the chem, custom-made words are harder to learn and are disconnected from the culture, semantics and the actual usage patterns in the game.

Do not make up a yet another `-esh` suffix because it is really stupid and lacks soul.

Upvoting the change.

**Review by stylemistake — APPROVED** [review]

**Comment by TemporalOroboros** [comment]
I am slightly tempted to make reagents change their name depending on who's looking at them. Chemists would see chems as representing their structure and/or recipies. The rest of medical would see them as representing how they can be used as medicine. Everyone else would go by some standard name. Perhaps medical would see the standard names by default and have a hotkey or action button to toggle between the styles of viewing... maybe the medical names would be the default for medical people. Perhaps a preference?

**Comment by evilcyborgmrt** [comment]
Multiver next please. Come on it is literally just charcoal

**Comment by stylemistake** [comment]
Charcoal with unusual properties, but still charcoal.

**Comment by lordpidey** [comment]
Regarding Sal-Glu's name, and Miner's Salve's name not fitting the nomenclature:

Saline Glucose's primary purpose is fixing bloodloss.  Brute+burn is a side benefit.

Miner's salve is meant to be a bad medicine... that can be made entirely on the mining station.  (Grind iron sheet, plasma sheet, and a twinkie).  Think home-remedy medicine.  It makes sense that a home remedy wouldn't fit the naming convention.

**Comment by ExcessiveUseOfCobblestone** [comment]
Ah yes charcoal which means it purges toxin and scales with multiple chems is more intuitive to people who have 0 medicinal knowledge than a set suffix you know to mean toxin healer and multi which wow look at that!

On the topic of RC, my chems are much easier to put in than say penteTic, oxandrOlone, and salIcYlic acid because they work on a convention. The hardest part would be the root since half the chem shares its name with several others versus any previous system.

I don't care what suffix you use but if it's intended to be made by the dispenser I expect it to abide by the convention. I'm willing to make an exception to this for synthflesh since I appeared to have made a mistake in middle grounding it instead of outright replacing the name like I did with others since I couldn't bother making a convention for a singular chem (we don't call convermol perf anymore and everyone lived).

**Comment by PKPenguin321** [comment]
@ExcessiveUseOfCobblestone Before you close this, I would like you to record yourself saying "instabitulari" five times fast and upload it

**Comment by ExcessiveUseOfCobblestone** [comment]
I don't like the mashed up suffix I've said this several times even before this PR. I agree it's hard to write and say.

I have just wrote that I'm making this an exception.

**Comment by PKPenguin321** [comment]
Do it anyways, it sounds real funny

**Comment by Electronics** [comment]
Ohhh is _that_ what that is‽

**Comment by craigcalef** [comment]
One or the other but having to call it "Synthflesh (instabitulari)" in plumbing devices where you enter free form text (filters, reaction chambers) is the worst.

**Pull request merged**